man/deepsea.1

```diff
@@ -0,0 +1,78 @@
+.TH deepsea 1 "DeepSea CLI man page"
+.SH NAME
+DeepSea CLI
+
+.SH SYNOPSIS
+deepsea [--version] [--help] [-l <log_level> | --log-level=<log_level>]
+        [--log-file=<path>]
+        <command> [<args>]
+
+.SH DESCRIPTION
+DeepSea CLI is a command-line-interface tool to run and visualize DeepSea's
+stages execution progress.
```

Comment:
is a command-line tool for running DeepSea stages and visualizing their execution progress.